packages/components/src/components/hds/tooltip-button/index.ts

```diff
@@ -6,29 +6,33 @@
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
 
-export const PLACEMENTS = [
-  'top',
-  'top-start',
-  'top-end',
-  'right',
-  'right-start',
-  'right-end',
-  'bottom',
-  'bottom-start',
-  'bottom-end',
-  'left',
-  'left-start',
-  'left-end',
-];
+import type { Props as TippyProps } from 'tippy.js';
+import { HdsTooltipPlacementValues } from './types.ts';
 
-export default class HdsTooltipIndexComponent extends Component {
+export const PLACEMENTS: string[] = Object.values(HdsTooltipPlacementValues);
+
+export interface HdsTooltipIndexSignature {
+  Args: {
+    extraTippyOptions: Omit<TippyProps, 'placement' | 'offset'>;
```

Comment:
Note: since placement and offset will always get overwritten on line 53, I figure we just omit them from the type to avoid confusion.